src/lib/libegl.js

```diff
@@ -507,6 +507,8 @@ var LibraryEGL = {
   eglGetError: () => EGL.errorCode,
 
   // EGLAPI const char * EGLAPIENTRY eglQueryString(EGLDisplay dpy, EGLint name);
+  // The allocations in glGetString are cached and never freed.
```

Comment:
```suggestion
  // The allocated strings are cached and never freed.
```
(or fix the function name)